addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -394,13 +394,30 @@ export class Wysiwyg extends Component {
         }
 
         const getYoutubeVideoElement = async (url) => {
-            const { embed_url: src } = await this._serviceRpc(
-                '/web_editor/video_url/data',
-                { video_url: url },
-            );
-            const [savedVideo] = VideoSelector.createElements([{src}]);
-            savedVideo.classList.add(...VideoSelector.mediaSpecificClasses);
-            return savedVideo;
+            if (URL.canParse(url)) {
```

Comment:
Let's remove the nesting!!
```suggestion
            if (!URL.canParse(url)) {
                return;
            }
```